Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -33,6 +34,8 @@ public class VenmoClient {
     static final String EXTRA_USERNAME = "com.braintreepayments.api.EXTRA_USER_NAME";
     static final String EXTRA_RESOURCE_ID = "com.braintreepayments.api.EXTRA_RESOURCE_ID";
 
+    static final String LOGGING_TAG = "Braintree";
```

Comment:
```suggestion
    static final String LOGGING_TAG = "Braintree SDK";
```
nit: take or leave. Also I could see us moving this constant to a `Core` file one day.

Reply:
Update and moved to a LoggingUtils class 👍